app/api/helpers/system_mails.py

```diff
@@ -88,7 +88,6 @@
         'message': (
             u"Your Account Has Been Created! Congratulations!" +
             u"<br/> <strong>Your login:</strong><br><strong>Email:</strong> {email}<br>"
-            u"<strong>Password:</strong> {password}<br><br> Please login and change your password as soon as possible"
```

Comment:
Removed since we are no longer sending the password on email. Please suggest if any other message to write here